context/ChainsContext/index.tsx

```diff
@@ -61,7 +66,13 @@ export const ChainsProvider = ({ children }: ChainsProviderProps) => {
     setChainsError(dispatch, chainItemsError);
 
     const loadedChain = getChain(chainItems);
-    setChain(dispatch, loadedChain);
+    const recentChains = getRecentChainNamesFromStorage();
```

Comment:
Why do we load recent chains? Is this used if non-empty? The `setChain(dispatch, loadedChain);` case does not seem to use recent chains

Reply:
Yes, that extra check is not needed. If `getChain()` returns empty chain it's because there are no recent chains